code/modules/surgery/organs/heart.dm

```diff
@@ -490,3 +490,29 @@
 	ethereal_heart.owner.gain_trauma(picked_trauma, TRAUMA_RESILIENCE_ABSOLUTE)
 	playsound(get_turf(ethereal_heart.owner), 'sound/effects/ethereal_revive.ogg', 100)
 	qdel(src)
+
+/obj/item/organ/second_heart
+	name = "second heart"
+	desc = "Wow, those lizards sure are full of heart."
+	icon_state = "second_heart"
+	zone = BODY_ZONE_CHEST
+	slot = ORGAN_SLOT_HEART_AID
+	healing_factor = STANDARD_ORGAN_HEALING
+	decay_factor = 2.5 * STANDARD_ORGAN_DECAY
```

Comment:
these hearts need a repair surgery, especially if they're going to decay at the same rate as normal hearts

Reply:
sure i guess i can do that later